lib/erb/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 class ERB
-  VERSION = '4.0.0'
+  VERSION = '4.1.0'
```

Comment:
Release management and adding a feature are two different things. Please refrain from doing multiple things in a single PR.

Reply:
changed back by resolving conflict